en/docs/references/about-this-release.md

```diff
@@ -98,6 +98,38 @@ The following new features and enhancements (introduced in WSO2 IS 6.0.0) are in
 
     [Learn more]({{base_path}}/deploy/configure-recaptcha/)
 
+-  **Tenant qualified URLs and tenanted sessions**
+
+    From IS 7.0.0 onwards, tenant qualified URLs will be enabled by default, which consistently qualifies every URL/ endpoint of WSO2 Identity Server with the tenant in a path parameter. This improves flexibility for tenant-wise sharding and branding compared to previous releases.
+
+    Additionally from IS 7.0.0 onwards, tenanted sessions will also be enabled allowing sessions and caches to be created in a tenanted space.
+
+    !!! info
+        It is strongly recommended to use WSO2 Identity Server with these two configs being enabled. However, if you need to disable any of these configs, follow the below steps.
+        
+        1. Add the following constraint to the "IDN_OAUTH_CONSUMER_APPS" table by executing the following sql command against the identity database (IDENTITY_DB).
+
+            ```
+            ALTER TABLE IDN_OAUTH_CONSUMER_APPS ADD CONSTRAINT UNIQUE_CONSUMER_KEY_CONSTRAINT UNIQUE (CONSUMER_KEY);
+            ```
+
+        2. Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory and add the following configuration.
+        
+            ```
+            [tenant_context]
+            enable_tenant_qualified_urls = false
+            enable_tenanted_sessions = false
+            ```
+
+        3. Restart WSO2 Identity Server.
+
+-   **OAuth client ID tenant unification**
+
+    OAuth consumer application client IDs are now tenant unique allowing same client ID to exist in multiple tenants. This feature requires enabling tenant qualified URLs and tenanted sessions which are by default enabled in IS 7.0.0 onwards.
+
+    !!! info
+        Once you start the identity server with tenant qualified URLs and tenanted sessions being enabled, you cannot disable them back as it could led to inconsistencies with this feature.
```

Comment:
This would have a significant effect. Hence, we would need to highlight this a bit more. @divyaamunugama wdyt?

Reply:
Shall we add this as a warning? 

!!! warning
    Once an identity server instance is started after enabling tenant-qualified URLs and tenanted sessions, you cannot disable these features. Proceed with caution.

Something like this would be sufficient IMO. WDYT?

Reply:
IMO what he meant is, adding this to some other main page other than about this release section to highlight better.

Reply:
Instead can we say something like?
```
If you're migrating from a previous version of Identity Server and have a requirement to disable any of these configs, contact the support for a proper migration.
```
We don't want any of the new users to disable these configs and should treat it as the behavior of IS. We should not confuse them with these additional configs. These configs are only important for those who migrate